.github/workflows/format.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.8
+          python-version: 3.9

```

Comment:
no strong reason. i guess we know 3.9 still works?

Reply:
Sounds good, I'm just wondering if we could make these changes easier next time.

Reply:
imo it's just a matter of convention, but I'd be happy to skip to the max python version we support whenever we drop support for the version used in CI. change every 3-4 releases instead of every release!